test/langtools/jdk/javadoc/lib/javadoc/tester/JavadocTester.java

```diff
@@ -1086,7 +1086,7 @@ public OutputChecker(String file) {
                 name = null;
                 content = null;
             } else {
-                name = file;
+                name = outputDir + "/" + file;
```

Comment:
I'm trying to understand the merit of this, since the output directory is often just `out` or `api` isn't it? Don't you want the test method name in there, for increased resolution?

Reply:
For test classes containing multiple test methods the output directory is actually the best way to identify which run of javadoc generated the output. If a test were to do multiple runs of javadoc with the same output directory I would certainly consider it a serious bug. Also, a single test method could easily do multiple runs of javadoc (I'm not sure we do that in our test suite, but I wouldn't be too surprised).  Adding the output directory disambiguates the file name which is already there, making it easy to locate the file itself and the code that generated it.

Reply:
OK; just verify it doesn't break existing tests!

Reply:
> I'm trying to understand the merit of this, since the output directory is often just `out` or `api` isn't it? Don't you want the test method name in there, for increased resolution?

I guess the simple filename of the output directory is often `out` or `api`, but the filename relative to the current directory is (or should be) unique.